packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -1765,6 +1691,9 @@ mod tests {
         static ref BOB_KEYPAIR: ChainKeypair = ChainKeypair::from_secret(&BOB).unwrap();
     }
 
+    const PRICE_PER_PACKET: u128 = 10000000000000000_u128;
+    const PATH_POS: u64 = 5_u64;
```

Comment:
Move this to `constants.rs` pls.

Reply:
These are constants in tests only, we do not have constant price nor position in the path :)